src/pages/PageAudioCall.tsx

```diff
@@ -1,15 +1,44 @@
 import React, { useState } from "react";
-import { useHistory, useParams } from "react-router-dom";
+import {
+  useHistory,
+  useLocation,
+  useParams,
+  RouteComponentProps,
+} from "react-router-dom";
+
 import { Helmet } from "react-helmet";
 
 import { useAudioSdk, useClient, useMicrophoneTrack } from "../hooks/audioSdk";
 import type { IMicrophoneAudioTrack } from "agora-rtc-sdk-ng";
+import {
+  EVENT_STREAM_PUBLISHED,
+  EVENT_STREAM_UNPUBLISHED,
+} from "../constants/events";
+import {
+  uniqueNamesGenerator,
+  Config,
+  adjectives,
+  colors,
+} from "unique-names-generator";
+
+const customConfig: Config = {
```

Comment:
Can you either:
1. give this var a better name (customConfig is kinda vague) and/or;
2. abstract this into a const in `constants` or;
3. inline this in your uniqueNamesGenerator call